custom-wiki-links.ts

```diff
@@ -0,0 +1,98 @@
+// Custom remark plugin to handle wiki links with proper path resolution
+
+export function customWikiLinks() {
+  const searchDirs = ["zk", "zk/Register"];
+  const baseDir = ".";
+  
+  function resolveLinkPath(link: string): string {
+    // Use Deno's file system API
+    try {
+      for (const searchDir of searchDirs) {
```

Comment:
1. First verify if `${link}.md` exists to avoid unnecessary searches.
2. Also considering situations `${link}/index.md` and `${link}/README.md`.